test/Microsoft.DotNet.Tools.Tests.Utilities/RepoDirectoriesProvider.cs

```diff
@@ -37,9 +37,14 @@ public static string RepoRoot
                 string directory = AppContext.BaseDirectory;
 #endif
 
-                while (!Directory.Exists(Path.Combine(directory, ".git")) && directory != null)
+                while (directory != null)
                 {
-                    directory = Directory.GetParent(directory).FullName;
+                    var gitDirOrFile = Path.Combine(directory, ".git");
+                    if (Directory.Exists(gitDirOrFile) || File.Exists(gitDirOrFile))
```

Comment:
Can you explain the need for the File.Exists call? I wasn't aware git could be signified in this way.